content/index.html

```diff
@@ -357,6 +333,13 @@ <h4 class="title">Latest Release
                                         <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Apache+CarbonData+1.3.1+Release"
                                            class="whatsnew" target="_blank">what's new</a>
                                     </div>
+                                    <div class="block-row">
+                                        <a href="https://dist.apache.org/repos/dist/release/carbondata/1.3.0/"
```

Comment:
All releases before 1.4.1 is nor working under the path https://dist.apache.org/repos/dist/release/carbondata/...

So such references can be removed